client/src/styles/colors.scss

```diff
@@ -1,6 +1,6 @@
 $white: #fff;
```

Comment:
@robinv85 I think we'll be removing `.scss` files right?

Reply:
yeah colors should go in theme constants now, but something like "white" may already be in there?

Reply:
Okay, point noted, every color goes into the constant files henceforth